workers/src/main/scala/workers/itc/ITCRequests.scala

```diff
@@ -42,7 +42,7 @@ object ITCRequests:
       ga.parTraverse(a => s.permit.use(_ => f(a)))
 
   // Wrapper method to match the call in ItcServer.scala
-  def queryItc[F[_]: Concurrent: Parallel: Logger](
+  def queryItc[F[_]: {Concurrent, Parallel, Logger as L}](
```

Comment:
Ouch. I admit I like the summoner pattern better rather than naming implicits.

Reply:
scalafmt doesn't like it either, I'll revert for now